examples/purchase-tester/src/main/java/com/revenuecat/purchasetester/OverviewFragment.kt

```diff
@@ -33,11 +33,11 @@ class OverviewFragment : Fragment(), OfferingCardAdapter.OfferingCardAdapterList
 
         binding.customerInfoLogoutButton.setOnClickListener {
             if (Purchases.sharedInstance.isAnonymous) {
-                findNavController().navigateUp()
```

Comment:
I switch from `navigateUp` to this since I added an option to clear the stack when going from the login fragment to the overview fragment. I thought it was strange to be able to go back to the login fragment with the back button, so I added that to disable it. Lmk if you have any concerns.